### DISCUSSION TITLE
change font size of context-menu

### DISCUSSION THREAD

**Discussion by ide5kllc** [microsoft/monaco-editor#3717] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

I know that it is possible to change the font-size of the EDITOR via:

monaco.editor.create(..., { fontSize: ... })

However, this does not seem to change the font size of the context-menu (from right click).

Is there a way to change the font size of the context-menu ?

**Comment by tamayika** [comment]
I digged VSCode codebase and found it is impossible now.

* There is no option to set CSS for context menu.
* Context menu is shown in [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM).
* Menu dom in Shadow DOM does not have [part](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/part) attribute, so we cannot apply css from outside of Shadow DOM.

@hediet
Please add part attribute with suitable value to [this dom](https://github.com/microsoft/vscode/blob/72def61faeb18afea67ca2cfe6a046354c52b343/src/vs/base/browser/ui/menu/menu.ts#L83).

**Comment by tamayika** [comment]
BTW, if you do not warry about ugly styling, please try to disable shadow dom with `useShadowDOM=false`.

ex.
```js
monaco.editor.create(document.getElementById('container'), {
	value: "function hello() {\n\talert('Hello world!');\n}",
	language: 'javascript',
	useShadowDOM: false,
});
```

```css
.monaco-menu {
    font-size: 20px;
}
```
